### ISSUE TITLE
Bakery

### ISSUE THREAD

**Issue by iSoNationwide** [ldtteam/MinecoloniesWiki#927]
Help Please

**Comment by Thodor12** [comment]
Please add a description of what is going wrong, you didn't state an issue in the title nor the description

**Comment by iSoNationwide** [comment]
Sorry I thought the description was filled, let me start over

I'm using a modpack with minecolonies in it for 1.16.5, the problem i have today is that the baker isn't doing anything, i.e. craftiung dough to make bread. I have tried using different worlds and modpacks and the result was the same. I tied looking at different forums online and troubleshooting that could be done, however, i don't think the problem lies with the baker but the bakery itself. The settings tab and task checklist is blank and no indication of having any avalibility to do anything. Even if I upgraded the bakery, hire different workers, or request anything, nothing shows. 

I was asking to see if maybe I can get a little help or an update, I'd be grateful.
